doc/release/1.16.0-notes.rst

```diff
@@ -47,6 +47,12 @@ Even when no elements needed to be drawn, ``np.random.randint`` and
 distribution. This has been fixed so that e.g.
 ``np.random.choice([], 0) == np.array([], dtype=float64)``.
 
+``linalg.lstsq`` now works with empty matrices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Previously, a ``LinAlgError`` would be raised when an empty matrix/empty
+matrices ("flat" or "skinny" ``a`` or "skinny" ``b``) is/are passed in.
```

Comment:
I'm not sure I've seen this terminology anywhere else before...

Reply:
Tall and skinny is used for data matrices with many rows. Would it be preferred to just use "0 rows" and "0 columns"?

Reply:
From a quick google, it looks like those terms are more used for things with a large and a small dimension, like `(3, 1000)` and `(1000, 3)`- rather than specifically for 0-size arrays. So yes, perhaps "with no columns or no rows" would be better.